src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-return-assign */
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const BookReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        action.payload,
+      ];
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.payload.id);
```

Comment:
- [ ] Next, we will need to refactor this return statement. Instead of `action.payload.id` it should be `action.payload`. This depends on how the removeBook action creator has been implemented. Currently, it returns `REMOVE_BOOK` action type and a `payload`.

```suggestion
      return state.filter((book) => book.id !== action.payload);
```

Reply:
Perfect, that was the problem, thank you very much for the nice code review!